pom.xml

```diff
@@ -474,6 +493,34 @@
           <linkJavadoc>true</linkJavadoc>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <configuration>
+          <append>true</append>
+        </configuration>
+        <executions>
+          <execution>
+            <id>agent-for-ut</id>
+            <goals>
+              <goal>prepare-agent</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>agent-for-it</id>
+            <goals>
+              <goal>prepare-agent-integration</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>jacoco-site</id>
+            <phase>verify</phase>
+            <goals>
+              <goal>report</goal>
+            </goals>
+          </execution>
+        </executions>
```

Comment:
interesting, what is the result?

Reply:
According to [`jacoco:report`](http://www.eclemma.org/jacoco/trunk/doc/report-mojo.html), it creates a code coverage report for tests of a single project in multiple formats (HTML, XML, and CSV).